src/_includes/site-header.html

```diff
@@ -1,54 +1,39 @@
-<header class="mobi-mart-header global-header fixed" role="banner" id="header">
-  <div id="skip-to-content"><a href="#main-content">Skip to Main Content</a></div>
-
-  <div class="branding">
-    <div class="header-organization-banner">
-      <a href="/">
-        <img
-          class="d-lg-none mobi-mart-logo"
-          src="/uploads/logo-sm.svg"
-          alt="California Integrated Travel Project (Cal-ITP)"
-        />
+<header role="banner" id="header">
+  <nav class="navbar navbar-expand-lg fixed-top bg-white main-navigation" aria-label="Navigation">
+    <div class="container">
+      <a class="navbar-brand" href="/">
+        <img class="d-lg-none mobi-mart-logo" src="/uploads/logo-sm.svg" alt="California Integrated Travel Project (Cal-ITP)" />
         <img
           class="d-none d-lg-inline mobi-mart-logo"
           src="/uploads/logo-lg.svg"
           alt="California Integrated Travel Project (Cal-ITP)"
         />
       </a>
-    </div>
-  </div>
-
-  <div class="mobile-controls">
-    <div class="mobile-control-group main-nav-icons">
       <button
-        id="nav-icon3"
-        class="mobile-control toggle-menu"
+        class="navbar-toggler collapsed"
+        type="button"
+        data-bs-toggle="collapse"
+        data-bs-target="#navbarsExample07"
+        aria-controls="navbarsExample07"
         aria-expanded="false"
-        aria-controls="navigation"
-        data-toggle="collapse"
-        data-target="#navigation"
+        aria-label="Toggle navigation"
       >
-        <span></span>
-        <span></span>
-        <span></span>
-        <span></span>
-        <span class="sr-only">Menu</span>
+        <span class="navbar-toggler-icon"></span>
       </button>
-    </div>
-  </div>
 
-  <div>
-    <nav
-      id="navigation"
-      class="main-navigation singlelevel auto-highlight no-icons nav"
-    >
-      <ul id="nav_list" class="top-level-nav">
-        {% include link-repeater.html
-             menu_items=site.navigation.header
-             list_class="nav-item"
-             anchor_class="first-level-link"
-        %}
-      </ul>
-    </nav>
-  </div>
+      <div class="navbar-collapse collapse" id="navbarsExample07" style="">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" data-bs-toggle="dropdown" aria-expanded="false">Products</a>
+            <ul class="dropdown-menu">
```

Comment:
I noticed that the dropdown menu has a lot of spacing between it and the nav-item. 

| Screenshot | Dev console |
|---|---|
| ![image](https://user-images.githubusercontent.com/25497886/218864623-eb8de60c-48ef-4230-a3ea-3de720c83405.png) | ![image](https://user-images.githubusercontent.com/25497886/218864736-b4f83fd5-c026-43c2-8d8e-7949b3c5d15b.png) |


Maybe we could add a style rule to bring them closer? Something like this?

| Screenshot | Dev console |
|---|---|
| ![image](https://user-images.githubusercontent.com/25497886/218864869-3f3e7c5e-8764-45d4-a1bf-ad0de03064b3.png) | ![image](https://user-images.githubusercontent.com/25497886/218864887-98efbaf1-51a3-4806-a6b6-ecf91a8e8ea6.png) |

Reply:
Agreed on the spacing for the nav! Otherwise, marking this as approved. Thank you!